docs/farming-&-staking/farming/advanced-cli/cli-farming-cluster.mdx

```diff
@@ -12,137 +13,420 @@ keywords:
+--in-connections
+--out-connections
+--pending-in-connections
+--pending-out-connections
 ```
-subspace-farmer cluster --nats-server nats://<NATS_IP>:4222 \
+</details>
+
+</details>
+
+<details open>
+<summary>Cache (Downloads sector data from DSN peers and serves it to the plotter.)</summary>
+
+:::caution Cache Size
+
+The Cache stores the blockchain's Archive History. For optimal performance, the cache must be large enough to hold this data locally. If the data isn't available in the cache, plotting may slow down as it has to be fetched from peers over the DSN. Currently, we recommend a minimum of 200GiB.
+
+The current Archive History Size can be found in the top cards on [Astral Block Explorer](https://astral.autonomys.xyz).
+
+:::
+
+```bash
+./subspace-farmer \
+    cluster \
+        --nats-server nats://<nats_ip>:4222 \
     cache \
-        path=/path/to/cache,size=SIZE
+        path=/path/to/cache,size=<size>
 ```
 
-Replace **`<NATS_IP>`** with your NATS server IP address. Provide the cache file path with **`path=`**, and specify the cache size. While an SSD is recommended, a hard disk can also be used.  200 GB is a good size to use for cache based on the current state of Mainnet.
+Replace the following placeholders:
 
-### Farmer
+- `<nats_ip>`: The IP address of your NATS server.
+- `/path/to/cache` and `<size>`: The path and size for storing the cache.
 
-Run the farmer component on each computer that will farm plots.
+<details>
+<summary>Common Paramaters</summary>
 
+Refer to `./subspace-farmer cluster cache --help` for an explanation of additional paramaters and their default values.
+
+```shell
+--cache-group
 ```
-subspace-farmer cluster --nats-server nats://<NATS_IP>:4222 \
-    farmer \
-        --reward-address <REWARD_ADDRESS> \
-        path=/path/to/farm,size=SIZE
+</details>
+
+</details>
+
+<details open>
+<summary>Plotter (Generates and uploads plotted sectors, downloading pieces from cache as needed)</summary>
+
+```bash
+./subspace-farmer \
+    cluster \
+        --nats-server nats://<nats_ip>:4222 \
+    plotter
 ```
 
-Replace **`<NATS_IP>`** with your NATS server IP address and provide your reward address, farm paths, and sizes. Optional parameters include **`--cpu-sector-encoding-concurrency`** or **`--cuda-sector-downloading-concurrency`**, **`--farming-thread-pool-size`**, **`--disable-farm-locking`**, and **`--create`**.
+Replace the following placeholders:
+
+- `<nats_ip>`: The IP address of your NATS server.
 
-### Plotter
+<details>
+<summary>Common Paramaters</summary>
 
-Finally, run the plotter component. At least one plotter is needed for plotting and replotting farms. It can be on the same computer as the farmer or on a separate machine.
+Refer to `./subspace-farmer cluster plotter --help` for an explanation of each option and its default values.
 
+```shell
+--cuda-sector-downloading-concurrency
 ```
-subspace-farmer cluster --nats-server nats://<NATS_IP>:4222 \
-    plotter
+</details>
```

Comment:
I probably wouldn't mention this to be completely honest, it should be needed to tweak with latest release. Also we have a similar option for ROCm build `--rocm-sector-downloading-concurrency`.